macros/base/start_print.cfg

```diff
@@ -325,9 +347,11 @@ gcode:
 
     # Load intial tool if needed
     {% if ercf_enabled %}
-        ERCF_RESET_STATS
-        ERCF_CHANGE_TOOL_STANDALONE TOOL={INITIAL_TOOL}
-        _DO_ERROR_ERCF_PAUSE
+        {% if ercf_reset_stats_on_start_print %}
+            ERCF_RESET_STATS
```

Comment:
Also can this be moved directly in the top of the `START_PRINT` macro before calling any module? I feel like this should not stay here as if for example someone tweak its `START_PRINT` sequence and do not include an extruder heating phase, this will never happens even if it's set to true in the `variables.cfg`

Reply:
ERCF_RESET_STATS is usefull for reset the statistics like numbers of swap in the current print... Personnaly i'll prefer to make a reset at the beginning of the print, so it cleans the previous statistics.

Reply:
thats personal choice. Ette und Moggieuk write statistics in a persistent file to be loaded on startup. I don't think they do that so the statistics are deleted on every print.

Reply:
> Also can this be moved directly in the top of the `START_PRINT` macro before calling any module? I feel like this should not stay here as if for example someone tweak its `START_PRINT` sequence and do not include an extruder heating phase, this will never happens even if it's set to true in the `variables.cfg`

You really do not own an ERCF.
ERCF loading means heating the extruder to at a configured value, which is at least 200 degrees which would make an issue with tap. Benoitone and i run this with ercf in exact this configuration successfully.

Reply:
I wouldn't change it.

Reply:
> You really do not own an ERCF. ERCF loading means heating the extruder to at a configured value, which is at least 200 degrees which would make an issue with tap. Benoitone and i run this with ercf in exact this start function successfully.

I was speaking about the resest stats only